ibis/backends/e6data/compiler.py

```diff
@@ -29,8 +32,11 @@ def rewrite_limit(_, **kwargs):
 
 
 @public
-class E6DataCompiler(MySQLCompiler):
+class E6DataCompiler(SQLGlotCompiler):
     __slots__ = ()
 
     dialect = E6data
     type_mapper = E6DataType
+
+    def visit_Equals(self, op, *, left, right):
+        return sge.EQ(this=left, expression=right)
```

Comment:
You are directly using SQLGlotCompiler
can just do 

```suggestion
        super().visit_Equals(op, left=left, right=right)
 ```